Extensions/Physics3Behavior/JsExtension.js

```diff
@@ -0,0 +1,3721 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
```

Comment:
Maybe a little text for know why we add this new physics v3.0.
For know what is the difference with v2.0.

Reply:
Yup, I am planning to include that as well, though  I need to decide which ones to put (that are significant and those that actually matter). Once again thanks a lot for reviewing a PR this big with so many typos.